sql/src/main/java/io/crate/action/sql/SQLOperations.java

```diff
@@ -280,7 +280,8 @@ public void execute(String portalName, int maxRows, ResultReceiver resultReceive
                     clearState();
                     ListenableFuture<?> result = portal.sync(planner, statsTables);
                     if (UNNAMED.equals(portal.name())) {
-                        portal.close();
+                        // simple statement must not allow 2nd bind() call after sync()
+                        portal.lock();
```

Comment:
I think this shouldn't result in an error if bind is called but instead it should implicitly close the portal on a 2nd bind